playwright-e2e/dsm/component/dropdown.ts

```diff
@@ -36,8 +40,14 @@ export default class Dropdown {
     return this.toLocator().locator('ul.dropdown-menu').locator('a').getByText(value, {exact: true});
   }
 
-  async getAllOptions(): Promise<Locator> {
+  async getDisplayedOptions<T extends MiscellaneousEnum | SamplesNavEnum | StudyNavEnum | StudyEnum>(): Promise<T[]> {
```

Comment:
`getAllOptions` is used in one test so you will need to update it to use new function.